### PR TITLE
Allow `now` to be passed as JS date instance.

### PR DESCRIPTION
**Pull request by JSAssassin**
Addresses issue #24

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # http-signature-zcap-verify ChangeLog
 
+## 8.2.0 - 2021-08-TBD
+
+### Changed
+- Allow `now` to be passed as a JS date instance.
```
Comment (dlongley) [inline CHANGELOG.md]:
```suggestion
- Allow `now` to be passed as a JS Date instance.
```

**Review by dlongley — APPROVED** [review]
LGTM, thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # http-signature-zcap-verify ChangeLog
 
+## 8.2.0 - 2021-08-TBD
+
+### Changed
```
Comment (dlongley) [inline CHANGELOG.md]:
```suggestion
### Added
```

File: CHANGELOG.md (hunk 1 above)
Comment (JSAssassin) [reply CHANGELOG.md]: Updated 9037ed9

File: CHANGELOG.md (hunk 2 above)
Comment (JSAssassin) [reply CHANGELOG.md]: Updated 9037ed9

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Comment by dmitrizagidulin** [comment]
Published as `8.2.0`

**Pull request merged**